### DISCUSSION TITLE
第3节课作业(6班)

### DISCUSSION THREAD

**Discussion by zhangzefeng92** [InternLM/Tutorial#211] [作业]
微信名称：捂上眼睛看世界
基础作业：https://www.toutiao.com/article/7322093756717711891/

**Comment by JackieGLQ** [comment]
微信名称：老高
基础作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#Yxtqdj88GoeKMixyn6HcPIDGnxc

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/764fd1f7-e47a-4812-8691-7c90d066e032](https://app.yinxiang.com/fx/764fd1f7-e47a-4812-8691-7c90d066e032)

**Comment by RalphGuo** [comment]
微信id：guos
![3](https://github.com/InternLM/tutorial/assets/58155843/d472fb29-548e-4120-aeaa-c2d04172b60e)
